themes/classic/templates/catalog/_partials/product-details.tpl

```diff
@@ -50,7 +50,11 @@
           <dl class="data-sheet">
             {foreach from=$product.features item=feature}
               <dt class="name">{$feature.name}</dt>
-              <dd class="value">{$feature.value}</dd>
+              {if $feature.id_feature === '4'}
```

Comment:
I'm totally agree with you.. some merchant wants this functionality apparently  (like price or discount..) http://forge.prestashop.com/browse/BOOM-1899